vercel.json

```diff
@@ -423,6 +423,146 @@
       "source": "/docs/guides/passwordless/social-sign-in",
       "destination": "https://www.ory.sh/docs/guides/social-signin/overview",
       "permanent": false
+    },
+    {
```

Comment:
How can we ensure that all required redirects do, in fact, work, and we did not forget one?

Reply:
I will crosscheck the deleted items with the redirects now.